### ISSUE TITLE
Allow custom docset urls

### ISSUE THREAD

**Issue by areina** [areina/helm-dash#6]
for example, for this:

https://raw.github.com/for-GET/know-your-http-well/master/docset/know-your-http-well.xml

**Comment by pronobis** [comment]
Yes, it would be very nice to be able to download docksets from: https://github.com/Kapeli/Dash-User-Contributions. Here's a nice discussion showing how to achieve that: https://github.com/zealdocs/zeal/issues/170

**Comment by kidd** [comment]
As a first approach, something like accepting a url should be enough, no?  I'm not aware if there are many non-official docsets. I have seen very few.

**Comment by pronobis** [comment]
Well, there is quite a bunch:
https://github.com/Kapeli/Dash-User-Contributions/tree/master/docsets

**Comment by kidd** [comment]
Oh yes, sorry I overlooked the first link. 
I'm working on a solution for this. hopefully it'll be up in a few days

**Comment by kidd** [comment]
@pronobis , there's a branch (currently still in development/testing) which implements that. 
if you have some time could you give it a shot?

branch is 'user-docsets', the command is `helm-dash-install-user-docset'. 
Bye!

**Comment by davidshepherd7** [comment]
I just gave the 'user-docsets' branch a try and it seems to be working perfectly (I installed two docsets from Dash-User-Contributions, added them to a local docsets hook and used them a few times).

One question: does there need to a be a different user-level function for installing user contributed docsets? Personally I don't care where the docsets come from, although it should be possible to find out if necessary.
